commands/initial/templates/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "{{ name }}",
   "version": "{{ version }}",
   "description": "{{ readmeTitle }}, an Angular library",
-  "main": "dist/index.js",
-  "jsnext:main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/{{ name }}.bundle.js",
+  "jsnext:main": "dist/{{ name }}.bundle.js",
+  "types": "dist/src/index.d.ts",
   "scripts": {
-    "build": "ngc",
+    "build": "webpack --config ./webpack/webpack.build.js",
```

Comment:
why'd you remove the `ngc` process? my understanding is that running it through the processor would make it so we don't have to distribute the TypeScript files (something I missed before).